commerce-account-web/src/main/java/com/liferay/commerce/account/web/internal/frontend/CommerceAccountAddressClayTable.java

```diff
@@ -92,15 +92,11 @@
 
 		Address address = (Address)model;
 
-		StringBundler sb = new StringBundler(7);
+		StringBundler sb = new StringBundler(3);
```

Comment:
Any time it's just 3 strings, we shouldn't use a `StringBundler`.... unless... we reset one of the strings, like we do here -- I think it's fine to do it this way. But if we ever have an sb of length 3 without resetting a string, we should change it to just concatenating strings.